### ISSUE TITLE
Накрутка оценок на карты и режимы 

### ISSUE THREAD

**Issue by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#67]
Есть возможность накрутить оценки на карты и режимы 
![Screenshot_2023-08-29-15-44-33-826_com pixel gun combats](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/142585063/cc48db15-0981-4b8a-b5d6-af0b66d8ff7a)

**Comment by forwork10** [comment]
Так, а где накрутка тут... Это вполне нормально что игроки ставят оценки...

**Comment by Kukushechka7-ezx** [comment]
Спт, ты долбаеб?) Даже скрины есть где ты говорил о этом) + 374 оценки за день, ты долбаеб?

**Comment by forwork10** [comment]
Какие ещё скрины... И эти все оценки появились не за один день, даже если посмотреть их айдишники - они старые... Оценки люди ставили давно, сейчас я просто в публикацию их поставил чтобы было видно их...

**Comment by Kukushechka7-ezx** [comment]
Бро, Я же все видел, как просто за день от неоткуда появились 300 оценок, да никто даже столкьо не чекает их,лол, это проверить легко накручены или нет

**Comment by forwork10** [comment]
Ок, раз вы видели как накручивают, то покажите видео, аналогично, если знаетк как проверить - проверьте и покажите нам.

**Comment by forwork10** [comment]
Да и не пахнет тут накруткой... Всего то 374... Зная мою популярность, это даже малость...

**Comment by Kukushechka7-ezx** [comment]
Ххахахах, ты долбаеб? Если бы ты был популярен твой зомби режим ( говно ) не тух бы на низу редимов

**Comment by Kukushechka7-ezx** [comment]
Сам щас сознался что накрутка, ахуеть) твой недо - зомби максимум по 10 человек собирает, дезмач и то больше

**Comment by forwork10** [comment]
покажи где я признался, и в чём... Опять ты из-за зависти начал поливать режим мой, интересно, почему только тебе он не нравится так)

**Comment by Kukushechka7-ezx** [comment]
Ты его считаешь каким то генеральный, какой то ахуенноп работой, лол, хотя он полная дрисня по сравнению с другими. Та же зб, зомби можно на ее уровне реализовать, но ты...

**Comment by Kukushechka7-ezx** [comment]
И опять, что ты высрал?? Бро, все понимают что твой зомби говно, по сравнению с нормальными режимами, а если судить как ты - тот же зек неа круче тебя, и келхун, у них режимов больше, и онлайна

**Comment by forwork10** [comment]
кто все? Ты вечно говоришь за всех, но по факту тебя не будет никто поддерживать, ты только посмотри на скрины![Screenshot_20230830-125014_VK.jpg](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/131859196/377e06f6-6cd5-40e0-92d8-b2a4ec10cc8a)

![Screenshot_20230830-125022_VK.jpg](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/131859196/27e4044b-e607-43bc-8c49-0c0f649e9d8c)

...

**Comment by forwork10** [comment]
![Screenshot_20230830-125242_blackbox(czk).jpg](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/131859196/8ec073ba-7d05-45a2-a5df-048a86c61ac7)

сейчас даже зашел посмотрел, нормальный онлайн, а когда выйдет обновление, где отведена вкладка для зомби режима, онлайн будет ещё больше)

**Comment by Kukushechka7-ezx** [comment]
А разве нету ее?) Ты тупоц! И чё, а кто они такие, чтобы оценивать МЕНЯ? Я круче чем они все

**Comment by forwork10** [comment]
эта версия есть, но она только внутренняя, тебя оценивают другие люди, ты просто не умеешь критику принимать. Да и ты просто полный идиот.... Это можно легко аргументировать.

**Comment by forwork10** [comment]
и ничем ты не круче других - ты ничего сейчас не имеешь.. абсолютно ничего.... ты полный 0. А внимания ты требуешь из-за своей ужасной "жизни". Все мы знаем в каких условиях ты выживаешь... И сам ты знаешь..

**Comment by forwork10** [comment]
во первых, хватит врать, я не писал такого... Да и трёх этажных домов не существует в моей городе... И опять ты начал говорить какой ты "крутой" а ты сначала покажи всё это...

**Comment by forwork10** [comment]
Вечно ты на меня переводишь всё, я говорю о тебе вообще-то...

**Comment by Kukushechka7-ezx** [comment]
Бро, а ты можешь свои хорошие работы показать?

**Comment by forwork10** [comment]
опять ты про меня... темболее я показывал тебе

**Comment by Kukushechka7-ezx** [comment]
Что показывал?) Говно - зомби и скрипт из инета для Майнкрафта?

**Comment by forwork10** [comment]
вот и вспоминай что я показывал. Ты ведь понимаешь насколько ты ничтожен по сравнению со мной, просто не признаешь этого... И не нужно сейчас отмазываться даже. У тебя сейчас даже нет ничего чтобы показать ...

**Comment by Kukushechka7-ezx** [comment]
Ау дурочка) у тебя 1 зомби, неон бокс выдумка, паблопаркуо лол делал, а маньяка чилли, скрипт из инета)

**Comment by forwork10** [comment]
а теперь на каждый твой "аргумент" дай доказательства

**Comment by forwork10** [comment]
ты сейчас чтобы доказать свои слова, требуешь меня что-то сделать, ну вот теперь скажи, не клоун ли ты...

**Comment by forwork10** [comment]
держи строчку исходника как и просил:
#include<stdio.h>

**Comment by kkohno** [comment]
так, по накруткам - тут с ходу и не определишь... однако вконце матча, если игрок не оценивал вылезает оценка карты... в итоге если было комнаты 2 с картой и по 24 игрока в течении пары суток то много оценок обеспечено думаю...

**Comment by Kukushechka7-ezx** [comment]
Да вы просто посмотрите на его карту, режимы, не совпадение ли? Что на 1 месте всё? Ещё так ровно

**Comment by forwork10** [comment]
я популярный просто, у меня есть кто может поставить оценки мне.....

**Comment by kkohno** [comment]
дайте ID накрученных по Вашему мнению карт/режимов

**Comment by Kukushechka7-ezx** [comment]
888 555 666 это режимы

**Comment by forwork10** [comment]
и конечно же ты только мои режимы считаешь накрученными)) типо тебе похер что там режимы закладка бомбы, ареаварс, свалка - имеют такое же количество оценок... ухх как ты меня ненавидишь)) Я рад что довель тебя до такого.. дай тебе шанс донести на кого-то, ты обратишь внимание только на меня.

**Comment by forwork10** [comment]
ухх как ты меня ненавидишь)) Я рад что довель тебя до такого.. дай тебе шанс донести на кого-то, ты обратишь внимание только на меня.

**Comment by Kukushechka7-ezx** [comment]
Может, потому что они в строгом порядке расположены? И их айди похожи, это не совпадение

**Comment by forwork10** [comment]
Я могу создавать совпадения.. На то я и гений

**Comment by kkohno** [comment]
Похоже реально накрутка идет. К сведению приняли, будем что-то с этим делать, спасибо

**Comment by forwork10** [comment]
😎

**Comment by forwork10** [comment]
сомневаюсь что с этим можно что-то сделать, возможность накрутки существует везде, живые аккаунт достать не трудно, айпи адрес перед входом в акк тоже подделать можно с помощью впн... Можно только сделать какой-то безумный алгоритм, но он будет мешать обычным игрокам ставить оценки

**Comment by Kukushechka7-ezx** [comment]
Есть способ, потом спишемся в ВК расскажу, ок

**Comment by Kukushechka7-ezx** [comment]
Там кстати на зомби ещё накрутка идёт

**Comment by forwork10** [comment]
> Есть способ, потом спишемся в ВК расскажу, ок

Да что ты ему умного расскажешь... КАК ПЕРЕКРЫТЬ ПАКЕТЫ ДАННЫХ ДО ДОСТАВКИ НА СЕРВЕР?!)

**Comment by forwork10** [comment]
> Там кстати на зомби ещё накрутка идёт 

завидуешь

**Comment by rtemon** [comment]
_не по теме но... ебаный античит, где он? А его нету, у меня куча айди читов _

**Comment by rtemon** [comment]
_не по теме но... ебаный античит, где он? А его нету, у меня куча айди читов _6B1B700E77846F1A

**Comment by kkohno** [comment]
исправлено, проверьте
